### ISSUE TITLE
Tracking: Getting the Scala backend to work

### ISSUE THREAD

**Issue by rudi** [abstools/abstools#282]
Branch scala (https://github.com/abstools/abstools/tree/scala) contains a Scala backend.  I (@rudi) have not yet gotten it to work (but did not spend significant time on it); help is welcome.  Below are the instructions from @vlad-serbanescu, the backend author  (June 28, Slack, private communication):

```
  so you need to clone this: https://github.com/JaacRepo/JAAC
  switch to "push-mechanism" branch
  and move the following:
  : cd ../generated-sources/jabsc; rm -rf ABS/JUnit; rm -rf ABS/Meta; rm -rf ABS/Scheduler; mv * path/to/JAAC/src/test/
  run main and at least hello world should be fine
  I usually run it from intellij
  actually run "src/test/Module.Main.scala"
  where "Module" is the name of the abs file where the "{}" block is defined
```

**Comment by vlad-serbanescu** [comment]
I'm back, I will make time during this month, to merge master back into the backend and restart work on unit tests

**Comment by rudi** [comment]
Hi Vlad, welcome back!  I haven't followed up in the meantime, so any help / automatic compilation (preferrably the latter ;) ) is very welcome.

**Comment by vlad-serbanescu** [comment]
Yes the compilation of the generated code requires both a jar (the thread efficient scala library) and some (too many) steps to run. I will work on automating this run so you can either run the Main class or directly a generated jar.

**Comment by rudi** [comment]
Closing since the Java backend is getting to where Scala would be; only thing missing for erlang-like scalability is support for virtual threads.
